src/cpu/cpu.rs

```diff
@@ -0,0 +1,49 @@
+use std::fmt;
+use crate::cpu::registers::{Registers, R8};
+
+use crate::instructions::execute_instruction;
+use crate::memory::MemoryBus;
+
+pub struct CPU {
+	pub registers: Registers,
+	pub pc: u16,
+	pub bus: MemoryBus,
+}
+
+impl CPU {
+	fn step(&mut self) {
+		let instruction_byte = self.bus.read_byte(self.pc);
+		execute_instruction(self, instruction_byte);
+
+		self.pc = self.pc.wrapping_add(1); // Implement PC
```

Comment:
```suggestion
		self.pc = self.pc.wrapping_add(1); // TODO Implement PC
```
don't forget to write TODO when it's a TODO